pom.xml

```diff
@@ -188,5 +272,19 @@
     <system>GitHub</system>
     <url>https://github.com/fcrepo4-exts/fcrepo-java-client/issues</url>
   </issueManagement>
+  
+  <distributionManagement>
+  <repository>
+    <id>nexus</id>
+    <name>Releases</name>
+    <url>https://maven.lib.umd.edu/nexus/content/repositories/releases</url>
+  </repository>
+  <snapshotRepository>
+    <id>nexus</id>
+    <name>Snapshots</name>
+    <url>https://maven.lib.umd.edu/nexus/content/repositories/snapshots</url>
+  </snapshotRepository>
+</distributionManagement>
```

Comment:
This section should be removed -- the artifacts should not, by default, be pushed to a UMD maven repo.